packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.component.tsx

```diff
@@ -165,12 +165,14 @@ const CustomIdentifier: React.FC<{ patient: SearchedPatient; identifierName: str
 }) => {
   const identifier = patient.identifiers.find((identifier) => identifier.identifierType.display === identifierName);
   return (
-    <>
-      <Tag size="sm" className={styles.configuredTag} type="warm-gray" title={identifier.display}>
-        {identifier.identifierType.display}
-      </Tag>
-      <span className={styles.configuredLabel}>{identifier.identifier}</span>
-    </>
+    identifier && (
```

Comment:
definitely let me know if there's a better/"proper" way to do a null check within a React.FC... is it okay that this could return null?

Reply:
I love the explicitness of ternaries https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx